scripts/release-branch.sh

```diff
@@ -7,5 +7,7 @@ VERSION=$(node -p -e "require('./package.json').version")
 BRANCH=$(echo "$VERSION" | (IFS="."; read a b c && echo $a.$b-stable))
 echo "$BRANCH"
 git checkout -b "$BRANCH"
-git push origin master --follow-tags
 git push origin "$BRANCH" --follow-tags
+git checkout master
+npm version preminor
```

Comment:
Is there a `yarn` command for this? We should use yarn instead of npm where possible :D